ui/desktop/app/styles/app.scss

```diff
@@ -244,6 +244,12 @@
   }
 }
 
+.target-details-body {
+  .hds-segmented-group {
```

Comment:
Isn't this margin normally handled by the `.search-filtering` css class?

Reply:
Is there some template hierarchy thing happening here? I notice two `search-filtering` classes

<img width="617" alt="image" src="https://github.com/user-attachments/assets/a01565b2-c6c9-4c57-aee4-765552fa08ac" />

Reply:
Weird. Let me take a look.

Reply:
Why are two being applied though? It seems like we have some weird parent/child templates as I notice two rose layouts and two rose headers. Maybe this is something we should investigate later but I'm not sure I would expect that